src/index.html

```diff
@@ -5,8 +5,334 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__link">
+        <img
+          class="header__logo logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__list-item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__list-item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              data-qa="nav__hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a class="nav__link" href="#">Gadgets</a>
+          </li>
+          <li class="nav__list-item">
+            <a class="nav__link" href="#">Tablets</a>
+          </li>
+          <li class="nav__list-item">
+            <a class="nav__link" href="#">Photo</a>
+          </li>
+          <li class="nav__list-item">
+            <a class="nav__link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="list-assortment">
+      <div class="list-assortment__card card" data-qa="card">
+        <a class="card__image" href="/">
+          <img
+            class="card__imac"
+            src="images/imac.jpeg"
+            alt="imac"
+          >
+        </a>
+
+        <h2 class="card__title title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__code code">Product code: 195434</p>
+
+        <div class="card__reviews-block">
+          <div class="card__stars stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+      
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price-block">
+          <p class="card__price price">Price:</p>
+          <p class="card__value value">$2,199</p>
+        </div>
+
+        <div class="card__buy">
+          <a
+            class="buy"
```

Comment:
```suggestion
            class="card__buy-link"
```
And here. Please fix all cases

Reply:
not fixed yet